src/services/llms/gemini.py

```diff
@@ -23,6 +23,7 @@ def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) ->
         - Use GitHub Markdown in comments
         - Focus on bugs, security issues, and performance problems
         - IMPORTANT: NEVER suggest adding comments to the code
+        - Comment with language of human: {Config.HUMAN_LANGUAGE}
```

Comment:
This line introduces a potential security risk.  Using `Config.HUMAN_LANGUAGE` directly without proper validation or sanitization could lead to injection vulnerabilities if the `HUMAN_LANGUAGE` config value is not properly controlled.  Consider adding input validation and sanitization to prevent this.  For example, you might want to create a whitelist of allowed languages or escape any special characters before using this value in your application.  The current implementation is vulnerable to Cross-Site Scripting (XSS) attacks if the language is displayed directly to a user without appropriate escaping.